pkg/signup/signup_service.go

```diff
@@ -213,3 +239,25 @@ func (s *ServiceImpl) GetSignup(userID string) (*Signup, error) {
 
 	return signupResponse, nil
 }
+
+func (s *ServiceImpl) GetUserSignup(userID string) (*crtapi.UserSignup, error) {
```

Comment:
can't this just be inside of `UpdateUserSignup`? then when you call UpdateUserSignup you could just pass it a userid similar to what's done in the other handlers?